test/Jamfile.jam

```diff
@@ -16,8 +16,11 @@ if [ os.name ] = NT
 }
 
 project : requirements
-  <toolset>msvc:<define>_SCL_5SECURE_NO_WARNINGS
+  <toolset>msvc:<define>_SCL_SECURE_NO_WARNINGS
+  <toolset>msvc:<define>_CRT_SECURE_NO_DEPRECATE
   <target-os>windows:<define>WIN32_LEAN_AND_MEAN
+  <target-os>windows:<define>NOMINMAX
+  <target-os>windows:<define>_WIN32_WINNT=0x0501
```

Comment:
Why windows version in the 'test' Jamfile', that's why:

..skipped <p..\..\..\bin.v2\libs\process\test\msvc-14.0\debug\threading-multi>sparring_partner.exe for lack of <p..\..\..\bin.v2\libs\filesystem\build\msvc-14.0\debug\threading-multi>boost_filesystem-vc140-mt-gd-1_62.lib...
...skipped <p..\..\..\bin.v2\libs\process\test\msvc-14.0\debug\threading-multi>sparring_partner.pdb for lack of <p..\..\..\bin.v2\libs\filesystem\build\msvc-14.0\debug\threading-multi>boost_filesystem-vc140-mt-gd-1_62.lib...
compile-c-c++ ..\..\..\bin.v2\libs\process\test\msvc-14.0\debug\threading-multi\sub_launcher.obj
sub_launcher.cpp
Please define _WIN32_WINNT or _WIN32_WINDOWS appropriately. For example:
- add -D_WIN32_WINNT=0x0501 to the compiler command line; or
- add _WIN32_WINNT=0x0501 to your project's Preprocessor Definitions.
Assuming _WIN32_WINNT=0x0501 (i.e. Windows XP target).
...skipped <p..\..\..\bin.v2\libs\process\test\msvc-14.0\debug\threading-multi>sub_launch.exe for lack of <p..\..\..\bin.v2\libs\filesystem\build\msvc-14.0\debug\threading-multi>boost_filesystem-vc140-mt-gd-1_62.lib...
...skipped <p..\..\..\bin.v2\libs\process\test\msvc-14.0\debug\threading-multi>sub_launch.pdb for lack of <p..\..\..\bin.v2\libs\filesystem\build\msvc-14.0\debug\threading-multi>boost_filesystem-vc140-mt-gd-1_62.lib...
compile-c-c++ ..\..\..\bin.v2\libs\process\test\environment.test\msvc-14.0\debug\threading-multi\environment.obj